plugins/woocommerce/tests/e2e-pw/tests/merchant/create-simple-product.spec.js

```diff
@@ -112,7 +115,42 @@ test.describe.serial( 'Add New Simple Product Page', () => {
 			waitUntil: 'networkidle',
 		} );
 		await page.locator( '#title' ).fill( nonVirtualProductName );
+
+		await page
+			.frameLocator( '#content_ifr' )
+			.locator( '.wp-editor' )
+			.fill( productDescription );
+
 		await page.locator( '#_regular_price' ).fill( productPrice );
+
+		await page.locator( '.inventory_options' ).click();
+		await page.locator( '#_sku' ).fill( '11' );
```

Comment:
Just a note on style:  this test is already using a number of `page.locators()` but going forward we're trying to use more of the human-facing locators in Playwright -- like `page.getByRole()` or `page.getByPlaceholder()`.  As you run into `page.locators()` in tests, try to refactor them to one of the other options going forward (sometimes it's the best/only choice though!)

Reply:
I pushed the update in [bb797b1](https://github.com/woocommerce/woocommerce/pull/43710/commits/bb797b1710976fe071cee51eb072ca1ed7dcbea1) and I'll comment in the code with following question I have

Reply:
For the `#_sku` field, I couldn't find a `getBy*` locator that would serve better. If you have a chance, can you take a look and share suggestions? I would appreciate to learn if there's a better way of getting this field

Reply:
@Stojdza  You can use `await page.getByLabel( 'SKU', { exact: true } ).fill( 'Test' );` for SKU field.

Reply:
This won't work for the text field, because it doesn't have a label itself: 

<img width="973" alt="Screenshot 2024-01-17 at 17 44 32" src="https://github.com/woocommerce/woocommerce/assets/7116222/9b9bb4f3-a5f3-4e90-a805-059e2170441e">